Conflux.API.Tests/Services/ProjectServiceTests.cs

```diff
@@ -112,4 +161,126 @@ public async Task UpdateProjectAsync_ShouldUpdateExistingProject()
+        Assert.NotNull(projectResult.Project);
+        Assert.Equal(projectId, projectResult.Project.Id);
+        Assert.Equal(testProject.Title, projectResult.Project.Title);
+        Assert.Equal(ProjectResultType.Success, projectResult.ProjectResultType);
+        Assert.Equal(projectResult.Project.People[0].Id, testPerson.Id);
+        Assert.Equal(projectResult.Project.People[0].Name, testPerson.Name);
+    }
+
+    [Fact]
+    public async Task AddPersonToProjectAsync_ShouldReturnNull_WhenProjectDoesNotExist()
+    {
+        // Arrange
+        ConfluxContext context = await Arrange();
+        ProjectService projectService = new(context);
+        
+        Guid projectId = Guid.NewGuid();
+        Guid personId = Guid.NewGuid();
+        
+        // Act
+        ProjectResult projectResult = await projectService.AddPersonToProjectAsync(projectId, personId);
+        
+        // Assert
+        Assert.NotNull(projectResult);
+        Assert.Null(projectResult.Project);
+        Assert.Equal(ProjectResultType.ProjectNotFound, projectResult.ProjectResultType);
+    }
+
+    [Fact]
+    public async Task AddPersonToProjectAsync_ShouldReturnProject_WhenPersonDoesNotExist()
+    {
+        // Arrange
+        ConfluxContext context = await Arrange();
+        ProjectService projectService = new(context);
+        
+        Guid projectId = Guid.NewGuid();
+        Guid personId = Guid.NewGuid();
+        
+        // Insert a test project
+        Project testProject = new()
+        {
+            Id = projectId,
+            Title = "Test Title",
+            Description = "Test Description",
+            StartDate = new DateOnly(2023, 1, 1),
+            EndDate = new DateOnly(2023, 12, 31)
+        };
+        
+        context.Projects.Add(testProject);
+        await context.SaveChangesAsync();
+        
+        // Act
+        ProjectResult projectResult = await projectService.AddPersonToProjectAsync(projectId, personId);
+        
+        // Assert
+        Assert.NotNull(projectResult);
+        Assert.NotNull(projectResult.Project);
+        Assert.Equal(projectId, projectResult.Project.Id);
+        Assert.Equal(testProject.Title, projectResult.Project.Title);
+        Assert.Equal(ProjectResultType.PersonNotFound, projectResult.ProjectResultType);
+        Assert.Null(projectResult.Project.People[0]);
+    }
+
+    [Fact]
+    public async Task AddPersonToProjectAsync_ShouldReturnProject_WhenPersonAlreadyExists()
+    {
+        
+    }
+
+    /// <summary>
+    /// This is a method that replaces the complete arrange part of all tests
+    /// since it the same for all tests in this class.
+    /// </summary>
+    /// <returns>The context</returns>
+    private async Task<ConfluxContext> Arrange()
+    {
+        var options = new DbContextOptionsBuilder<ConfluxContext>()
+            .UseNpgsql(_postgres.GetConnectionString())
+            .Options;
+
+        await using ConfluxContext context = new(options);
```

Comment:
Door het toevoegen van await using regel je expliciet de lifetime van de context. In dit geval is dit niet goed, want de lifetime van de database is alleen in de scope van de ze functie. Dus wat er gebeurt is als de functie de context returned wordt de context direct gedisposed 🙈 

```suggestion
        ConfluxContext context = new(options);
```